test/OpenTelemetry.Exporter.OpenTelemetryProtocol.FuzzTests/Generators.cs

```diff
@@ -0,0 +1,236 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Diagnostics;
+using System.Diagnostics.Metrics;
+using FsCheck;
+using FsCheck.Fluent;
+using OpenTelemetry.Exporter.OpenTelemetryProtocol.Implementation;
+using OpenTelemetry.Logs;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Resources;
+
+namespace OpenTelemetry.Exporter.OpenTelemetryProtocol.FuzzTests;
+
+internal static class Generators
+{
+    public static readonly ActivitySource TestActivitySource = new("Fuzz.ActivitySource", "1.0.0");
+
+    private static readonly Gen<ActivityStatusCode> ActivityStatusCodes = Gen.Elements(
+    [
+        ActivityStatusCode.Unset,
+        ActivityStatusCode.Ok,
+        ActivityStatusCode.Error,
+    ]);
+
+    private static readonly Gen<LogRecordSeverity> LogRecordSeverities = Gen.Elements(
+    [
+        LogRecordSeverity.Debug,
+        LogRecordSeverity.Error,
+        LogRecordSeverity.Fatal,
+        LogRecordSeverity.Info,
+        LogRecordSeverity.Trace,
+        LogRecordSeverity.Unspecified,
+        LogRecordSeverity.Warn,
+    ]);
```

Comment:
Candidate for separate PR:
```suggestion
    private static readonly Gen<LogRecordSeverity> LogRecordSeverities = Gen.Elements(
    [
        LogRecordSeverity.Debug,
        LogRecordSeverity.Error,
        LogRecordSeverity.Fatal,
        LogRecordSeverity.Info,
        LogRecordSeverity.Trace,
        LogRecordSeverity.Unspecified,
        LogRecordSeverity.Warn,
        (LogRecordSeverity)int.MaxValue,
    ]);
```

Reply:
I'll do those "bad values" as a separate PR, as FsCheck should be able to generate "interesting" values without having to explicitly deal with `MaxValue`, `-1` etc I believe.

Reply:
Addressed in this PR anyway, as I needed to make some changes locally to use the unused method to generate log records.